emulators/spim/files/patch-spim_Makefile.diff

```diff
@@ -1,41 +1,19 @@
---- spim/Makefile.orig	2009-01-14 22:29:50.000000000 -0800
-+++ spim/Makefile	2009-01-14 23:24:24.000000000 -0800
-@@ -51,16 +51,16 @@
- # Path of directory that contains documentation:
- DOC_DIR = ../Documentation
+--- Makefile-ori.txt	2018-09-07 07:57:28.000000000 +0800
++++ Makefile	2018-09-10 16:13:56.000000000 +0800
+@@ -66,13 +66,13 @@
  
--
-+prefix = /usr/local
  
  # Full path for the directory that will hold the executable files:
--BIN_DIR = /usr/bin
+-BIN_DIR = $(DESTDIR)/usr/bin
 +BIN_DIR = $(prefix)/bin
  
  # Full path for the directory that will hold the exception handler:
--EXCEPTION_DIR = /usr/lib/spim
+-EXCEPTION_DIR = $(DESTDIR)/usr/share/spim
 +EXCEPTION_DIR = $(prefix)/share/spim
  
  # Full path for the directory that will hold the man files:
--MAN_DIR = /usr/share/man/en
+-MAN_DIR = $(DESTDIR)/usr/share/man/man1
 +MAN_DIR = $(prefix)/share/man/man1
  
  
- # If you have flex, use it instead of lex.  If you use flex, define this
-@@ -174,11 +174,14 @@
- 	rm -f spim spim.exe *.o TAGS test.out lex.yy.c y.tab.c y.tab.h y.output
  
- install: spim
--	install -D spim $(BIN_DIR)/spim
--	install -D -m 0444 $(CPU_DIR)/exceptions.s $(EXCEPTION_DIR)/exceptions.s
-+	install -d $(DESTDIR)$(BIN_DIR)
-+	install -m 0755 spim $(DESTDIR)$(BIN_DIR)/spim
-+	install -d $(DESTDIR)$(EXCEPTION_DIR)
-+	install -m 0644 $(CPU_DIR)/exceptions.s $(DESTDIR)$(EXCEPTION_DIR)/exceptions.
```

Comment:
You didn't put a `.s` at the end here. You're installing the wrong filename.

Reply:
This problem is caused by the way Github displays the code, you need to review the two diff files manually and you will find that this is not a typo caused by me.

Reply:
Ah, my mistake. You removed that entire section. It was like that before you changed it.